sdk/opendp/whitenoise/evaluation/dp_verification.py

```diff
@@ -454,15 +463,15 @@ def main(self):
 
         d1_query = "SELECT Role, Segment, COUNT(UserId) AS UserCount, SUM(Usage) AS Usage FROM d1.d1 GROUP BY Role, Segment"
         d2_query = "SELECT Role, Segment, COUNT(UserId) AS UserCount, SUM(Usage) AS Usage FROM d2.d2 GROUP BY Role, Segment"
-        dp_res, acc_res, utility_res, bias_res = dv.dp_groupby_query_test(d1_query, d2_query, plot=False, repeat_count=1000)
+        dp_res, acc_res, utility_res, bias_res = dv.dp_groupby_query_test(d1_query, d2_query, plot=False, repeat_count=200)
 
         # Powerset Test on SUM query
         query_str = "SELECT SUM(Usage) AS TotalUsage FROM "
-        dp_res, acc_res, utility_res, bias_res = self.dp_powerset_test(query_str, plot=False, repeat_count=1000)
+        dp_res, acc_res, utility_res, bias_res = self.dp_powerset_test(query_str, plot=False, repeat_count=100)
 
         # WhiteNoise-Core Test
-        root_url = subprocess.check_output("git rev-parse --show-toplevel".split(" ")).decode("utf-8").strip()
-        test_csv_path = os.path.join(root_url, "service", "datasets", "evaluation", "PUMS_1000.csv")
+        pums_1000_dataset_path = os.path.join(os.path.dirname(__file__), '../../../../service', "datasets", "evaluation", "PUMS_1000.csv")
```

Comment:
these should be inputs in the test instead of part of the library

Reply:
Removed main function which had these tests